lib/ecto/migration.ex

```diff
@@ -496,20 +499,20 @@ defmodule Ecto.Migration do
   @doc ~S"""
   Returns an index struct that can be used on `create`, `drop`, etc.
 
-  Expects the table name as first argument and the index fields as
-  second. The field can be an atom, representing a column, or a
-  string representing an expression that is sent as is to the database.
-
-  Indexes are non-unique by default.
+  Expects the table name as the first argument and the index field(s) as
+  the second. The fields can be atoms, representing columns, or strings,
+  representing expressions that are sent as-is to the database.
 
   ## Options
 
-    * `:name` - the name of the index. Defaults to "#{table}_#{column}_index"
-    * `:unique` - if the column(s) is unique or not
-    * `:concurrently` - if the index should be created/dropped concurrently
-    * `:using` - configures the index type
-    * `:prefix` - prefix for the index
-    * `:where` - the conditions for a partial index
+    * `:name` - the name of the index. Defaults to "#{table}_#{column}_index".
+    * `:unique` - indicates whether the index should be unique. Defaults to
+      `false`.
+    * `:concurrently` - indicates whether the index should be created/dropped
+      concurrently.
+    * `:using` - configures the index type (for PostgreSQL).
```

Comment:
Is `(for PostgreSQL)` accurate? [There is a test on the MySQL side that test `using: :hash`](https://github.com/elixir-ecto/ecto/blob/master/test/ecto/adapters/mysql_test.exs#L1092-L1096), and [documentation of the MySQL seem to accept `USING`.](https://dev.mysql.com/doc/refman/5.7/en/create-index.html)

Reply:
Made this change; thanks for the catch!